api/controllers/auth.js

```diff
@@ -50,54 +63,65 @@ const reset = async (req, res) => {
 const resetApprovementPassword = async (req, res) => {
   try {
     const { linkId } = req.params;
+
     if (!linkId) {
       throw new errors.ResetPasswordApproveError('Empty params');
     }
-    const UserId = await services.resetPasswordApprove({
+
+    const userId = await services.resetPasswordApprove({
       linkId: decodeURIComponent(linkId),
     });
-    if (UserId) {
+
+    if (userId) {
       res.status(200).send({
         status: 200,
-        message: { content: 'Enter New Password', UserId },
+        message: { content: 'Enter New Password', userId },
       });
     }
-
   } catch (e) {
     throw new errors.ResetPasswordError(e.message);
   }
 };
 
 const resetPassword = async (req, res) => {
   const { linkId, password, passwordConfirm } = req.body;
+
   if (!linkId || !password || !passwordConfirm) {
     throw new errors.ResetPasswordError('Empty params');
   }
+
   try {
     const validation = await passwordsSchema.validate({
       password,
       passwordConfirm,
     });
+
     const answer =
       validation && (await services.resetPassword({ password, linkId }));
+
     res.status(200).send(answer);
   } catch (e) {
     throw new errors.ResetPasswordError(e.message);
   }
 };
 
 const changePassword = async (req, res) => {
-  const errors = validateAuth(req.body);
-  if (!isEmpty(errors)) {
-    return res.status(400).json(errors);
+  const { password } = req.body;
```

Comment:
i offer you not to use dectructurization for 1 field

Reply:
Resolved